extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+![States](figures/states.png)
+
+### Root Tiles
+
+Multiple tilesets using this extension can be combined by using parent tileset referring to one or more external tilesets using `3DTILES_implicit_tiling` as the children of the root tile.
+
+### Jump Buffer
+
+The jump buffer is a buffer created at runtime to allow efficient traversal and tile lookup functions. The jump buffer is a bijection between a tile's Morton index and its index in the level.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_implicit_tiling Tileset JSON extension
+
+Specifies the Tileset JSON properties for the 3DTILES_implicit_tiling.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**boundingVolume**|`object`|A bounding volume that encloses the tileset.|☑️ Yes|
+|**tilingScheme**|`string`|A string describing the tiling scheme used within the tileset|☑️ Yes|
+|**subdivision**|`object`|An object containing high level information about the subdivision buffer|☑️ Yes|
+|**content**|`object`|An object containing high level information about the content buffer. This may be omitted if no tiles in the tileset contain content.|No|
+|**metadata**|`object`|An object containing high level information about the metadata buffer. This may be omitted if no tiles in the tileset contain metadata.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
+|**tileExtension**|`string`|The extension applied to each tile in the tileset.|No|
+|**tilesetExtension**|`string`|The extension applied to each implicit external tileset in the tileset.|No|
+
+Additional properties are not allowed.
+
+---
+
+
+### boundingVolume 
+
+A bounding volume that encloses the tileset.  Exactly one `box`, `region` or `cell` property is required.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**box**|`number` `[12]`|An array of 12 numbers that define an oriented bounding box. The first three elements define the x, y, and z values for the center of the box.  The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.  The next three elements (indices 6, 7, and 8) define the y axis direction and half-length.  The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.|No|
+|**region**|`number` `[6]`|An array of six numbers that define a bounding geographic region in EPSG:4979 coordinates with the order [west, south, east, north, minimum height, maximum height]. Longitudes and latitudes are in radians, and heights are in meters above (or below) the WGS84 ellipsoid.|No|
+|**cell**|`number` `[3]`|An array of 10 numbers that defines a geodesic quadrilateral cell. The first 8 elements are 4 pairs of EPSG:4979 coordinates in radians, and the last 2 elements are heights in meters above the WGS84 ellipsoid.|No|
```

Comment:
Offline we talked about `cell` being allowed in the general `boundingVolume` schema (as long as `3DTILES_implicit_tiling` is preset) so that base tilesets can also use `cell` bounding volumes. Is that still the intent?

Reply:
At the moment, it's only a part of the extension since we may decide to change the way this whole concept works in order to be a better fit for S2.